### PR TITLE
Set clusterIPRange param as part of creating kubetest bash deployer

### PR DESCRIPTION
**Pull request by shyamjvs**
Ref https://github.com/kubernetes/kubernetes/issues/55246

/cc @krzyzacy

### CODE REVIEW

File: kubetest/bash.go [hunk 1]
```diff
@@ -25,39 +25,34 @@ import (
...
-		cmd = "../federation/hack/e2e-internal/e2e-up.sh"
+		script = "../federation/hack/e2e-internal/e2e-up.sh"
 	} else {
-		cmd = "./hack/e2e-internal/e2e-up.sh"
+		script = "./hack/e2e-internal/e2e-up.sh"
 	}
-	return finishRunning(exec.Command(cmd))
+	cmd := exec.Command(script)
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, fmt.Sprintf("CLUSTER_IP_RANGE=%s", b.clusterIPRange))
```
Comment (krzyzacy) [inline kubetest/bash.go]: I remember we pushed the env for other purposes, does the actual e2e test need this env var set?

Comment (shyamjvs) [reply kubetest/bash.go]: Yes, the env var is used by kube-up script. And we need this in order to allow overriding default that would be used by kube-up.

Comment (shyamjvs) [reply kubetest/bash.go]: For e2e test, we do use env var [here](https://github.com/kubernetes/kubernetes/blob/e1de2ad50776454386be42160e1fbf6291ba8cd9/test/e2e/framework/test_context.go#L351) to override the default value. But that's not needed anymore as this PR will instead pass that value through the flag.

I'll follow up with a PR in k/k removing that code piece.

Comment (krzyzacy) [reply kubetest/bash.go]: xref https://github.com/kubernetes/kubernetes/pull/51343#issuecomment-325302544, I didn't see the flag passes the iprange flag to ginkgo being set? Is there another PR somewhere?

Comment (shyamjvs) [reply kubetest/bash.go]: Good catch - I thought I was setting `o.clusterIPRange` as well. Fixed it now.
The flag is being set here - https://github.com/shyamjvs/test-infra/blob/532911eb77af3e122f7ad790f0b11f4d096db715/kubetest/e2e.go#L40
whose value is coming from `o.clusterIPRange`, which in turn is being set inside `newBash()` now.

**Comment by krzyzacy** [comment]
/lgtm
send a image bump PR next?

**Pull request merged**